1209/8787/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: MVMC-lab_M128_V3
+owner: MVMC-lab
+license: MIT License
+site: https://gitlab.com/MVMC-lab
+source: https://gitlab.com/MVMC-lab/usb_transmit_program_system
```

Comment:
This looks like the link for the software. What is the link for the specific hardware info?

Reply:
I've changed to hardware PCB layout project link.
It contains software link in Readme.md file in hardware project link.

Reply:
Thank you for the link update. Please also include licenses in the repos.

Reply:
Yes, the repos need the license as well. GitLab still shows them as all rights reserved.

Why do both PIDs have the same source repo? Is it the same hardware?

Reply:
They base on the similar architecture but not the same boards.
I've update them to 2 repo, both of them have software links in Readme. (Also license

Thank you!

Reply:
> Yes, the repos need the license as well. GitLab still shows them as all rights reserved.
> 
> Why do both PIDs have the same source repo? Is it the same hardware?

They base on the similar architecture but not the same boards.
I've update them to 2 repo, both of them have software links in Readme. (Also license

Thank you!

Reply:
The two repo links look good. They still need `LICENSE` files in them though. They shouldn't have this `All rights reserved` portion on them.

![Screen Shot 2021-01-09 at 5 55 37 PM](https://user-images.githubusercontent.com/52649/104112461-0233de80-52a4-11eb-9928-7e9293db9851.png)

Reply:
Licenses in both projects were updated!

Reply:
> The two repo links look good. They still need `LICENSE` files in them though. They shouldn't have this `All rights reserved` portion on them.
> 
> ![Screen Shot 2021-01-09 at 5 55 37 PM](https://user-images.githubusercontent.com/52649/104112461-0233de80-52a4-11eb-9928-7e9293db9851.png)

Licenses in both projects were updated!

Reply:
`LICENSEs` in both projects were updated!